modules/storage/storage-account/main.bicep

```diff
@@ -19,6 +19,102 @@ param isZoneRedundant bool = false
 @description('Storage Account Type. Use Zonal Redundant Storage when able.')
 param storageAccountType string = isZoneRedundant ? 'Standard_ZRS' : 'Standard_LRS'
 
+@description('Toggle to enable or disable Blob service of the Storage Account.')
+param enableBlobService bool = false
+
+@description('Properties object for a Blob service of a Storage Account.')
+param blobProperties blobServiceProperties = {}
+
+@description('Name of a blob container to be created')
+param blobContainerName string = 'default'
+
+@description('Properties object for a Blob container of a Storage Account.')
+param blobContainerProperties blobServiceContainerProperties = {}
+
+@description('Array of role assignment objects that contain the \'roleDefinitionIdOrName\', \'principalId\' and \'resourceType\' as \'storageAccount\' or \'blobContainer\' to define RBAC role assignments on that resource. In the roleDefinitionIdOrName attribute, you can provide either the display name of the role definition, or its fully qualified ID in the following format: \'/providers/Microsoft.Authorization/roleDefinitions/c2f4ef07-c644-48eb-af81-4b1b4947fb11\'')
+param roleAssignments array = []
+
+@description('Toggle to enable or disable versioning for Blob service of the Storage Account. Used only if enableBlobService is set to true.')
+type isBlobVersioningEnabled = false | true
+
+type blobServiceProperties = {
+  changeFeed: changeFeed?
+  containerDeleteRetentionPolicy: containerDeleteRetentionPolicy?
+  cors: cors?
+  deleteRetentionPolicy: deleteRetentionPolicy?
+  isVersioningEnabled: isBlobVersioningEnabled?
+  lastAccessTimeTrackingPolicy: lastAccessTimeTrackingPolicy?
+  restorePolicy: restorePolicy?
+}
+
+type changeFeed = {
+  enabled: bool
+  retentionInDays: int
+}
+
+type containerDeleteRetentionPolicy = {
+  allowPermanentDelete: bool
+  days: int
+  enabled: bool
+}
+
+type cors = {
+  corsRules: [
+    {
+      allowedHeaders: [
+        'string'
+      ]
+      allowedMethods: [
+        'string'
+      ]
+      allowedOrigins: [
+        'string'
+      ]
+      exposedHeaders: [
+        'string'
+      ]
+      maxAgeInSeconds: int
```

Comment:
and constraints for the min and max value of this int.

Reply:
There is no default min/max as per docs https://learn.microsoft.com/en-us/azure/templates/microsoft.storage/storageaccounts/blobservices?pivots=deployment-language-bicep#corsrule 

But we can agree mutually on setting something if it makes sense!
<html>
<body>
<!--StartFragment-->

maxAgeInSeconds | Required if CorsRule element is present. The number of seconds that the client/browser should cache a preflight response. | int (required)
-- | -- | --


<!--EndFragment-->
</body>
</html>